### PR TITLE
Add preload script to conf

### LINKED ISSUE #2121: Add preload scripts to configuration
Users can provide a `--preload` script to their dask-worker or dask-scheduler commands to launch custom code during execution.  This is pragmatic when you want to extend Dask with plugins, but somewhat annoying to do every time.

We might add two entries to the configuration `distribued.scheduler.preloads` and `distributed.worker.preloads` that are lists of scripts or imports to pass to the Worker and Scheduler on startup.  This should be the default within the `worker.py::WorkerBase` and `scheduler.py::Scheduler` constructors.

This was first suggested, I think, by @lesteve

### PR DESCRIPTION
**Pull request by guillaumeeb**
Fixes #2121.

Still trying to dive inside codebase, so I pick up some simple issues. Two questions for this one:
- Did not find any test about preload scripts. Don't know how to build one... Should we?
- For Scheduler, preload script are only loaded in cli.dask_scheduler, so I made the modification here, is this OK?

### CODE REVIEW

File: distributed/cli/dask_scheduler.py [hunk 1]
```diff
@@ -125,6 +126,10 @@ def del_pid_file():
                           scheduler_file=scheduler_file,
                           security=sec)
     scheduler.start(addr)
+    if not preload:
+        preload = dask.config.get('distributed.scheduler.preloads')
+    if not preload_argv:
+        preload_argv = dask.config.get('distributed.scheduler.preloads-argv')
```
Comment (mrocklin) [inline distributed/cli/dask_scheduler.py]: Thoughts on preload vs preloads?  I'm not sure if we should pluralize this word, given the single-ness of the `--preload` CLI keyword.

Comment (guillaumeeb) [reply distributed/cli/dask_scheduler.py]: Fine by me to go with preload, I'll update the PR accordingly.

**Comment by guillaumeeb** [comment]
>  I think that it would be good to move this within the Scheduler and Worker classes

For Worker this is already the case. For Scheduler, there seems to be some reason about it, thus I'm not sure this is a trivial change. I did not find a simple way to do it when I looked at it.

**Comment by mrocklin** [comment]
Test failures here seem unrelated.  See https://github.com/dask/distributed/pull/2331

**Comment by mrocklin** [comment]
Sorry for the delay on this.  TLS issues were fixed in other PRs.  Things are green now.  Merging.

**Comment by guillaumeeb** [comment]
No problem, I was planning to look at the tests you mentioned to see if I can add something. But this seems simple enough to merge it as it is, hopefully I did not messed up with default kwargs :slightly_smiling_face:

**Pull request merged**